### PR TITLE
Bump grafana-formula version

### PR DESCRIPTION
**Pull request by witekest**
New version adds SUSE 15.3 to the list of supported versions.

### CODE REVIEW

File: grafana-formula/grafana-formula.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 22 08:31:16 UTC 2021 - Witek Bedyk <witold.bedyk@suse.com>
+
+- Version 0.4.0
+- Add SUSE 15.3 to supported versions
```
Comment (juliogonzalez) [inline grafana-formula/grafana-formula.changes:5]: `SUSE Linux Enterprise 15 SP3` instead of `SUSE 15.3`,  maybe?

Comment (witekest) [reply grafana-formula/grafana-formula.changes]: Added `openSUSE` as well.

File: grafana-formula/grafana-formula.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 22 08:31:16 UTC 2021 - Witek Bedyk <witold.bedyk@suse.com>
+
+- Version 0.4.0
+- Add SLES 15 SP3 and openSUSE 15.3 to supported versions
```
Comment (juliogonzalez) [inline grafana-formula/grafana-formula.changes:5]: Hehe, sorry to be so picky, but can you then write `openSUSE Leap 15.3`? :-)

Other than that, this looks good.

**Review by juliogonzalez — APPROVED** [review]

File: grafana-formula/grafana-formula.changes (hunk 2 above)
Comment (witekest) [reply grafana-formula/grafana-formula.changes]: done

**Pull request merged**